openfasoc/generators/temp-sense-gen/tools/parse_rpt.py

```diff
@@ -16,3 +19,26 @@
         raise ValueError("LVS failed!")
     else:
         print("LVS is clean!")
+        
+with open('test.json', 'r') as file:
+    data = json.load(file)
+print('Found .json config file...')
+
+module_name = data.get("module_name", "default")
+
+work_dir = "./work/"
+
+if (os.path.exists(work_dir) == 0):
+    raise ValueError("work directory does not exist!")
+else:
+    filename = work_dir + module_name
+    for file in (filename + ".gds", filename + ".spice", filename + ".v", filename + ".def", filename + "_pex.spice", filename + ".sdc"):
```

Comment:
Avoid hard-coding inside the logic itself. I would suggest to use list here to store the file extensions so that in future it can be updated with more file extensions if needed. Refer to how the drc report and lvs report file name are stored in the very beginning.

Reply:
Resolved in [https://github.com/idea-fasoc/OpenFASOC/pull/251/commits/f3057569c854d497f2a7f01566381986b3810d44](url)